Makefile

```diff
@@ -64,8 +64,15 @@ build-sol:
 	git submodule update --init --recursive
 	forge build --names --force
 
+build-vyper:
+	poetry run python vyper_contracts/scripts/compile.py
```

Comment:
why not just `poetry run vyper -f abi,bytecode,bytecode_runtime yourFileName.vy` (maybe check the flags)

Reply:
It does work but it prints the result in the cli instead of generating build files. That is why added the script in  `compile.py` to put the result in build folder so that can be used later while deploying.

Not sure if there are other ways to do it